### PR TITLE
A new article: Go through sites's subpages but do not reload the web chat

### PR DESCRIPTION
**Pull request by dsocha**

### CODE REVIEW

File: data/authors.yml [hunk 1]
```diff
@@ -11,4 +11,9 @@
...
   twitter: PureCloud_Dev
   bio: Tim has been with Interactive Intelligence/Genesys since 2007 and has focused on designing and developing customer-facing integrations and customizations.
-  
\ No newline at end of file
+- email: dariusz.socha@genesys.com
+  name: Dariusz Socha
+  title: Sr Impl & Dev Engineer
+  company: Genesys
+  twitter: 
+  bio: Dariusz has joined to Interactive Intelligence/Genesys in 2015. He is a PS application developer in the EMEA.
```
Comment (idemtidem) [inline data/authors.yml]: can skip "to"..."Dariusz has joined Interactive Intelligence/Genesys in 2015."

File: source/2017-07-18-embedding-web-chat/index.html.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: Go through sites's subpages but do not reload the web chat
+date: 2017-07-18
+tags: chat, webchat, ACD chat
+author: dariusz.socha@genesys.com
+---
+
+Following [the PureCloud documentation](https://developer.mypurecloud.ie/api/webchat/) you can choose one of two methods of embedding a chat widget into a customer’s webpage. The first method opens the widget in a pop-up window but the second one renders it in a dedicated div element inside the webpage. The main browser window can easily cover the pop-up chat, what isn't desired from a web visitor’s experience point of view. The widget embedded directly in the page seems to be much better solution as it can be rendered on the top most div element and it gives much more possibilities. Using top most div you can show the widget as a sliding or draggable page element in accordance to the latest web design trends.
```
Comment (idemtidem) [inline source/2017-07-18-embedding-web-chat/index.html.markdown]: Some suggestions, feel free to take or leave them. I'd just personally add more business-oriented cases into the intro before diving into the implementation.

Change "but" to "and": "The first method opens the widget in a pop-up window and the second one renders it in a dedicated div element inside the webpage."

I'd elaborate on the pop up item, i.e. 
"The pop-up chat option has its challenges, however. Often, the web visitor forgets about the pop-up as it gets covered up by the main browser window and when they are finally connected to an agent they are no longer responding. This leads to a frustrating experience and also wastes agent's time."

Then for the widget itself, I'd also provide more business-oriented explanation. "The widget embedded directly in the page seems to be much better solution as it stays front and center in front of the customer's eyes" and then explain about the top most div element.

File: source/2017-07-18-embedding-web-chat/index.html.markdown [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+author: dariusz.socha@genesys.com
+---
+
+Following [the PureCloud documentation](https://developer.mypurecloud.ie/api/webchat/) you can choose one of two methods of embedding a chat widget into a customer’s webpage. The first method opens the widget in a pop-up window but the second one renders it in a dedicated div element inside the webpage. The main browser window can easily cover the pop-up chat, what isn't desired from a web visitor’s experience point of view. The widget embedded directly in the page seems to be much better solution as it can be rendered on the top most div element and it gives much more possibilities. Using top most div you can show the widget as a sliding or draggable page element in accordance to the latest web design trends. 
+
+### All good?
+The webpage is in the background, the chat widget is on the top. The web visitor is chatting to an agent and once he is clicking a link in order to navigate to another subpage then… the spell was broken. The browser is reloading the page, the widget is rendered from scratch, the visitor is going to the ACD queue again and… ‘Hi, how can I help you today?’ - a completely new agent is asking.
```
Comment (idemtidem) [inline source/2017-07-18-embedding-web-chat/index.html.markdown]: " once he is clicking a link" - should be "once he **clicks**"
"the spell was broken" - I'd keep it in present tense "the spell **is** broken"
"The browser is reloading the page" - should be "the browser **reloads** the page"
"the visitor is going to the ACD queue again" - I'd change to "the visitor is **sent** to the ACD queue again"

File: source/2017-07-18-embedding-web-chat/index.html.markdown [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+### All good?
+The webpage is in the background, the chat widget is on the top. The web visitor is chatting to an agent and once he is clicking a link in order to navigate to another subpage then… the spell was broken. The browser is reloading the page, the widget is rendered from scratch, the visitor is going to the ACD queue again and… ‘Hi, how can I help you today?’ - a completely new agent is asking. 
+
+### What’s wrong? 
+Nothing. It works as it was combined with the page. The thing is to reload the main page but do not reload the chat widget. It's easy. Make your index.html file a kind of frameset and put your webpage and the chat widget into separate iFrames.
```
Comment (idemtidem) [inline source/2017-07-18-embedding-web-chat/index.html.markdown]: I did not quite follow this. It seems like here you need to introduce it as..."The good news is that you can avoid this issue by following the solution described in this article. Once you do this, the web chat widget works as if it were combined with the page"" or similar and then go into the solution as you did.

File: source/2017-07-18-embedding-web-chat/index.html.markdown [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+### All you need are a few HTML tags:
+~~~html
+<html>
+ <body>
+  <iframe style="position:absolute; width:100%; height:100%; top:0px; right:0px; bottom:0px: left:0px; z-index:1;" src="{PATH TO YOUR WEBPAGE}"></iframe>
+  <iframe style="position:absolute; top:20px; right:20px; height:400px; width:400px; z-index:9999999;" src="{PATH TO A CHAT HTML FILE}"></iframe>
+ </body>
+</html>
+~~~
+Having the webpage embedded in a separate iFrame you can go through subpages of your site and it doesn't reload the chat session. It gives much better expirience for web visitors.
```
Comment (idemtidem) [inline source/2017-07-18-embedding-web-chat/index.html.markdown]: I'd change to "**When you have** the webpage embedded..."

typo - should be **experience**, or you can even elaborate further what the experience is, i.e. "Web visitors will be able to chat with your agents while they are browsing the site"

**Comment by dsocha** [comment]
I added a few updates related to comments.

**Pull request merged**